src/Frame/Frame.cc

```diff
@@ -1773,17 +1803,32 @@ bool Frame::FitImage(const CARTA::FittingRequest& fitting_request, CARTA::Fittin
                 return false;
             }
 
+            float image_std;
+            std::vector<CARTA::StatsType> required_stats = {CARTA::StatsType::Sigma};
+            bool per_z = false;
+            std::map<CARTA::StatsType, std::vector<double>> stats_map;
```

Comment:
This should probably be `unordered_map`, but I see that it's `map` in the existing code. There are a few other places where we're using `map`; this should probably be raised in a separate PR (I don't think there are any cases where we actually need an ordered map -- ensuring that we use `unordered_map` may improve performance, but `map` uses less memory, so perhaps this warrants further discussion).

Reply:
@confluence Yes, `GetRegionStats` requires `map`. Looks like we don't set up a long list of stats type, so it's fine to use `map` for now.
Do you prefer I add a TODO in `GetRegionStats` or file a code maintenance issue?

Reply:
Filed #1295.